mera/mera_pymer4.py

```diff
@@ -14,6 +15,9 @@ def run_mera(
     compute_site_term: bool = True,
     mask: pd.DataFrame = None,
     verbose: bool = True,
+    verbose_warnings: bool = True,
```

Comment:
I think `raise_warnings` instead of `verbose_warnings` is better. Verbose to me implies extra stuff that can generally be ignored, which doesn't really apply to these warnings

Reply:
Maybe thats just me though